omero/sysadmins/version-requirements.txt

```diff
@@ -34,9 +34,7 @@ Operating systems
 
 * MacOS X
 
-  * [5.3] 10.9 dropped, 10.10 deprecated, 10.11 recommended, 10.12 upcoming
-  * [5.4] 10.10 dropped, 10.11 deprecated, 10.12 recommended, 10.13 upcoming
-  * Rationale: MacOS X is typically suited only to client use, not serious server deployment, so limit testing accordingly.
+  * MacOS X is typically suited only to client use, not serious server deployment, so limit testing accordingly.
```

Comment:
I would rather not mixed the 2.
We were listing "conventional" clients. There is no mention of tablets for example. This is now heavily used for example by students using teaching.